Rakefile

```diff
@@ -5,7 +5,7 @@ begin
     config.exclude_labels = %w{duplicate question invalid wontfix wont-fix modulesync skip-changelog}
```

Comment:
can you please add `github_actions` to the exclude list? We try to keep those automatic PRs for actions out of the changelog.

Reply:
done

Reply:
you also need to regenerate the changelog

Reply:
no changes

Reply:
run `rm -rf /tmp/github-changelog-*` and try again

Reply:
after `github_action` seems to be a newer label. after manually adding it for old PRs its now acknowledged